packages/store/addon/-private/system/record-arrays/adapter-populated-record-array.js

```diff
@@ -74,14 +74,46 @@ export default RecordArray.extend({
 
     return store._query(this.modelName, query, this);
   },
+});
+
+if (RECORD_ARRAY_MANAGER_IDENTIFIERS) {
+  /**
+    @method _setIdentifiers
+    @param {StableRecordIdentifier[]} identifiers
+    @param {Object} payload normalized payload
+    @internal
+  */
+  AdapterPopulatedRecordArray.prototype._setIdentifiers = function _setIdentifiers(identifiers, payload) {
```

Comment:
Seems safer to extend here

Reply:
Done!  I think the original intention was to work on the object directly instead of subclassing but this works too!  Aligns closer to the semantics of defining objects in this file as well.  

One question though - When it comes to switching this file over to TS though, would we modify the prototype or subclass?